doc/deps.md

```diff
@@ -91,31 +94,33 @@ other commands below, unless *explicitly* specified!**
 ### Thrift
 CodeCompass needs [Thrift](https://thrift.apache.org/) which provides Remote
 Procedure Call (RPC) between the server and the client. Thrift is not part of
-the official Ubuntu 16.04 LTS and 18.04 LTS repositories, but you can download it
-and build from source:
+the official Ubuntu 16.04 LTS and 18.04 LTS repositories, but you can download
+it and build from source:
 
 ```bash
-# Thrift may require yacc and flex as dependency:
-sudo apt-get install byacc flex
-
 # Download and uncompress Thrift:
 wget "http://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=thrift/0.12.0/thrift-0.12.0.tar.gz" \
   -O thrift-0.12.0.tar.gz
 tar -xvf ./thrift-0.12.0.tar.gz
 cd thrift-0.12.0
 
-./configure --prefix=<thrift_install_dir> --with-python=NO --with-php=NO
-# Thrift can generate stubs for many programming languages. The configure script
-# looks at the development environment and if it finds the environment for a
-# given language then it'll use it. For example in the previous step npm was
-# installed which requires NodeJS. If NodeJS can be found on your machine then
-# the corresponding stub will also compile. If you don't need it then you can
-# turn it off: ./configure --without-nodejs.
+# Ant is required for having Java support in Thrift.
+sudo apt-get install ant
```

Comment:
Do we need Java support in Thrift?

Reply:
Yes, because search parser is written in Java and Thrift is used for internal communication with C++ code.

Reply:
Yeah, right.

Reply:
Yeah, right. But still, I don't have the `ant` package installed on my computer, and managed to compile Thrift with Java support.